filesystem.md

```diff
@@ -106,3 +107,35 @@ The `Storage` facade may be used to interact with any of your configured disks.
 #### Delete A Directory
 
 	Storage::deleteDirectory($directory);
+
+<a name="custom-filesystems"></a>
+## Custom Filesystems
+In order to set up your custom filesystem you'll need to create a Service Provider (For example, `DropboxFilesystemServiceProvider`. In its `boot` method, you'll want to inject an instance of the `Illuminate\Contracts\Filesystem\Factory` contract, and then call the `extend` method of the injected instance.
```

Comment:
Missing the closing paren on the "For example"; also, it should be "for example", no capital.